docs/subnets/customize-a-subnet.md

```diff
@@ -152,6 +161,53 @@ The fields `nonce`, `timestamp`, `extraData`, `gasLimit`, `difficulty`, `mixHash
 set to match the `gasLimit` set in the `feeConfig`. You do not need to change any of the other genesis
 header fields.
 
+`nonce`, `mixHash` and `difficulty`
+
+These are remnant parameters from Proof of Work systems.
+For Avalanche, these don't play any relevant role, so you should just leave them as: 
+
+- `nonce` as `0x0`.
+- `mixHash` as `0x0000000000000000000000000000000000000000000000000000000000000000`.
+- `diffuculty` as `0x0.
+
+`timestamp`
+
+The timestamp of the creation of the genesis block. This is commonly set to `0x0`.
+
+`extraData`
+
+Optional extra data that can be included in the genesis block. This is commonly set to `0x`.
+
+`gasLimit`
+
+The total amount of gas that can be used in a single block. It should be set to
+the same value as in the [fee config](#fee-config). The value `e4e1c0` is
+hexadecimal and is equal to `15,000,000`.
+
+`coinbase`
+
+Refers to a special transaction included in each block of the blockchain. It is the first 
+transaction within the block and serves as the mechanism through which the block 
+reward is generated. It is usually set
+to `0x0000000000000000000000000000000000000000` for the genesis block.
```

Comment:
This sounds like `coinbase` itself is the transaction. `coinbase` is an recipient address for block rewards (in our case only tx fees) that could be collected by block producers. In other words this is the address of whomever produced the block (i.e miner)

Reply:
You're right! it does sound like that. For Avalanche, I understand there is no a single "miner" for each block, so for every block other than genesis, how does this works? what address its registered there?

Reply:
Blocks are still proposed/built by single entities (miner/producer/proposer etc). 

The block structure of Coreth and Subnet-EVM are very similar to Eth blocks. Coinbase is always a part of coreth and subnet-evm blocks and represent the rewards address of whomever created the block. In Coreth this address has to be a blackhole address meaning that we burn all fees. However in Subnet-EVM there is an optional config (`allowFeeRecipients`) that can re-enable distributing these fees to the block producers through the coinbase. So in Subnet-EVM the coinbase can still represent block producer addresses.